core/psh/pshapp/pshapp.c

```diff
@@ -53,7 +53,13 @@
 
 
 /* Special key codes */
-enum { kUp = 1, kDown, kRight, kLeft, kDelete, kHome, kEnd };
+enum { kUp = 1,
```

Comment:
Looks like a weird formatting to me. I'm not sure about our current formatting guidelines but I'd either keep the original code or move `kUp = 1` to the next line:
```c
enum {
    kUp = 1,
```

Reply:
Hmm, clang-format formatted it like that. I left the original formatting for this line and separate changes into 2 commits: formatting and history fix.

Reply:
Yeah, our clang-format configuration doesn't seem to handle `enum` well. Something to consider in the future. Making an exception here is fine for now.